.github/workflows/e2e-report.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         id: get-run-id
         run: |
           if [ "${{ inputs.use-branch }}" == "true" ]; then
-            E2E_RUN_ID=$(gh run list -w test-e2e.yml -e workflow_dispatch -s success -b $GITHUB_REF_NAME --json databaseId --jq ".[0].databaseId" --repo $GITHUB_REPOSITORY)
+            E2E_RUN_ID=$(gh run list -w test-e2e.yml -s success -b $GITHUB_REF_NAME --json databaseId --jq ".[0].databaseId" --repo $GITHUB_REPOSITORY)
```

Comment:
not really related to deleting deploys, but this allow to generate test results report from runs on pull request (when label is applied) and not just on `workflow_dispatch`